src/OSPSuite.Infrastructure.Import/Services/IImportFromExcelTask.cs

```diff
@@ -100,13 +100,23 @@ private static IEnumerable<DataTable> dataTables(string fileName, string sheetNa
 
             while (reader.MoveToNextRow())
             {
-               dataTable.AddRowToDataTable(reader.CurrentRow);
+               var rowList = reader.CurrentRow;
+               rowList = removeColumnsWithoutHeader(rowList, dataTable.Columns.Count);
+
+               dataTable.AddRowToDataTable(rowList);
             }
 
             dataTablesList.Add(dataTable);
          }
 
          return dataTablesList;
       }
+
+      private static List<string> removeColumnsWithoutHeader(List<string> rowList, int headerCount)
+      {
+         if (rowList.Count > headerCount)
+            rowList = rowList.GetRange(0, headerCount);
```

Comment:
If I am understanding properly, you are stopping at the first header empty. Is this the expected behavior

Reply:
Not the the first empty header, we stop at the first element of a row that exceeds the number of headers